### PR TITLE
Strict null checks next 2

### PR DESCRIPTION
**Pull request by grahamlangford**
## What does this PR do?

- More strict null checks work

## Reviewer Tips

- Fixed an issue with the scanning tool that was messing up relative imports and preventing a lot of issues from being added

## Checklist

- [x] Designate a primary reviewer @mnholtz

### CODE REVIEW

File: src/pageEditor/starterBricks/contextMenu.ts [hunk 1]
```diff
@@ -131,20 +132,28 @@ async function fromExtension(
...
   const base = baseFromExtension(config, extensionPoint.definition.type);
   const extension = await extensionWithNormalizedPipeline(
     config.config,
     "action",
   );
 
+  assertNotNullish(
+    extensionPoint.metadata,
+    "Starter brick metadata is required",
+  );
```
Comment (grahamlangford) [inline src/pageEditor/starterBricks/contextMenu.ts:152]: @twschiller Should we be throwing a type error or handling if metadata is undefined here? I'll be handling all of the starter bricks as part of this PR

Comment (twschiller) [reply src/pageEditor/starterBricks/contextMenu.ts:152]: IIRC, we should be throwing an error. During load the Page Editor creates some placeholder metadata for the starter brick: https://github.com/pixiebrix/pixiebrix-extension/blob/fccd376cf7b8b15cce422ff6981840977bfd916d/src/pageEditor/starterBricks/base.ts#L203-L203

Comment (grahamlangford) [reply src/pageEditor/starterBricks/contextMenu.ts:152]: That's what I thought, but I wanted to confirm. Thanks

File: src/pageEditor/starterBricks/formStateTypes.ts
Comment (mnholtz) [inline src/pageEditor/starterBricks/formStateTypes.ts:1]: @grahamlangford can you help me understand why we're going to `undefined` here?

Comment (grahamlangford) [reply src/pageEditor/starterBricks/formStateTypes.ts:1]: See: https://github.com/pixiebrix/pixiebrix-extension/pull/8544/files/c72506da4b8b48e4d80ffe6d530f2c20129246fb#diff-9f3adfd38ce97176ee83c2d7cecb3f5b09e8ef4539f4521b098d403919b90011R50:~:text=function%20fromNativeElement(-,function%20fromNativeElement(,)%3B,-assertNotNullish(

I initially change the typing to `null`, but that would force major factory changes and led to some nasty/hairy test failures

**Review by mnholtz — APPROVED** [review]

**Pull request merged**